tvm_cli/README.md

```diff
@@ -334,7 +334,7 @@ List of available options:
 --access_key <ACCESS_KEY>                     Project secret or JWT in Evercloud (dashboard.evercloud.dev).
 --addr <ADDR>                                 Contract address.
 --async_call <ASYNC_CALL>                     Disables wait for transaction to appear in the network after call command.
---balance_in_tons <BALANCE_IN_TONS>           Print balance for account command in tons. If false balance is printed in nanotons.
+--balance_in_tons <BALANCE_IN_VMSHELLS>           Print balance for account command in VMSHELLs. If false balance is printed in VMSHELL.
```

Comment:
It's better to rename flag to `balance_in_vmshells`

Reply:
Ready